### ISSUE TITLE
ML - Training Script - Decision Trees

### ISSUE THREAD

**Issue by Mohammad4844** [COSC481W-2024Fall/MLite#8]
## User Story
As a user, I would like to train a decision tree model. I want to be able to go to my dataset, select a label, choose amongst some pair of hyperparameters, and train a Decision Tree model.

### Persona 
Existing users looking to train a model.

### Feature
Be able to run a training job through our app, specifically for a decision tree model.

### Business Value
Users can train a new model type from our app for datasets with categorical labels.

## Tasks

1. Script takes in input hyperparameters
2. Script loads the dataset
3. Script initializes a decision tree model & trains on the dataset
4. Script saves/uploads the trained model

## Acceptance Criteria
- Will NOT implement integration with our Rails app. Simply ML scripts to train a given model.

## Acceptance Tests

- Users can schedule a Decision Tree model training
- Training runs successfully
- Final model is saved in an appropriate location
